devicetypes/smartthings/zigbee-valve.src/zigbee-valve.groovy

```diff
@@ -39,7 +41,7 @@ metadata {
     }
 
     tiles(scale: 2) {
-        multiAttributeTile(name:"contact", type: "generic", width: 6, height: 4, canChangeIcon: true){
+        multiAttributeTile(name:"valve", type: "generic", width: 6, height: 4, canChangeIcon: true){
```

Comment:
Please leave this tile as "contact". There are local execution reasons that this is contact and not "valve".